src/generate/module/index_spec.ts

```diff
@@ -1,55 +1,49 @@
 import { join } from 'path';
 
-import { VirtualTree } from '@angular-devkit/schematics';
-import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+import { SchematicTestRunner, UnitTestTree } from '@angular-devkit/schematics/testing';
 import { getFileContent, createAppModule } from '@schematics/angular/utility/test';
 
 import { Schema as ModuleOptions } from './schema';
-import { createEmptyProject, toNgModuleClassName } from '../../utils';
+import { toNgModuleClassName, createEmptySharedProject, createEmptyNsOnlyProject } from '../../utils';
 import { DEFAULT_SHARED_EXTENSIONS } from '../utils';
 import { isInModuleMetadata } from '../../test-utils';
 
-xdescribe('Module Schematic', () => {
-  const path = 'app';
-  const sourceDir = 'app';
+describe('Module Schematic', () => {
   const name = 'foo';
   const project = 'test';
   const moduleClassName = toNgModuleClassName(name);
   const defaultOptions: ModuleOptions = {
     project,
     name,
-    path,
-    web: false,
-    nativescript: true,
   };
   const schematicRunner = new SchematicTestRunner(
     'nativescript-schematics',
     join(__dirname, '../../collection.json'),
   );
-  const getModulePath = (extension: string) => `/${sourceDir}/${path}/${name}/${name}.module${extension}.ts`;
+  const getModulePath = (extension: string) => `/src/app/${name}/${name}.module${extension}.ts`;
   const noExtensionModulePath = getModulePath('');
   const nsModulePath = getModulePath(DEFAULT_SHARED_EXTENSIONS.ns);
   const webModulePath = getModulePath(DEFAULT_SHARED_EXTENSIONS.web);
 
-  const getRoutingModulePath = (extension: string) => `/${sourceDir}/${path}/${name}/${name}-routing.module${extension}.ts`;
+  const getRoutingModulePath = (extension: string) => `/src/app/${name}/${name}-routing.module${extension}.ts`;
   const noExtensionRoutingModulePath = getRoutingModulePath('');
   const nsRoutingModulePath = getRoutingModulePath(DEFAULT_SHARED_EXTENSIONS.ns);
   const webRoutingModulePath = getRoutingModulePath(DEFAULT_SHARED_EXTENSIONS.web);
 
-  let appTree;
-  
-  beforeEach(() => {
-    appTree = new VirtualTree();
-    appTree = createAppModule(appTree);
-    appTree = createEmptyProject(appTree);
-  });
- 
+  let appTree: UnitTestTree;
+
+
+  // Structure proposal
   describe('when in ns-only project', () => {
+    beforeEach(() => {
+      appTree = createEmptyNsOnlyProject(project);
+    });
+
+    // tests
```

Comment:
This one as well.

Reply:
Removed! Perfectly descriptive comment though :trollface: